packages/data-store/src/types/contact/contact.ts

```diff
@@ -153,6 +153,23 @@ export type NaturalPerson = {
 export type NonPersistedNaturalPerson = Omit<NaturalPerson, 'id' | 'createdAt' | 'lastUpdatedAt'>
 export type PartialNaturalPerson = Partial<NaturalPerson>
 
+export type Student = {
```

Comment:
Why are we adding a Student Entity? That is not at all what we want. We discussed to be able to extend individuals with additional properties. We are not creating a product around students, so we should be flexible in supporting new individual types, without having to provide all explicit entities for these types.